framework/lib/components/badge/types.ts

```diff
@@ -0,0 +1,11 @@
+import { BadgeProps as ChakraBadgeProps } from '@chakra-ui/react'
+
+type BadgeSize = '2xs' | 'xs' | 'sm' | 'md'
+type BadgeVariant = 'solid' | 'outline' | 'subtle'
+type BadgeColorSchemes = 'gray' | 'mediatoolBlue' | 'blue' | 'red' | 'green' | 'orange' | 'yellow' | 'teal' | 'purple' | 'pink'
```

Comment:
We already have a type called `Color` in framework/lib/types/color. It does miss the 'teal' color tho so you could add it. A better solution might be to write something like this: `import { palette } from '@northlight/tokens'
const colorSchemes = keys(omit([ 'mono' ], palette))
type ColorScheme = typeof colorSchemes[number]
` - this way the ColorScheme type updates with the tokens